### DISCUSSION TITLE
convert `ZArray` to `YAXArray`

### DISCUSSION THREAD

**Discussion by kongdd** [JuliaIO/Zarr.jl#127] [Q&A] [answered]
Dear developers, 

Could you give an example about how to convert `ZArray` to `YAXArray` without writing into disk?

Thanks very much.

**Comment by felixcremer** [comment, marked as answer]
To construct an YAXArray you can call the constructor. `YAXArray(arr)`. This gives you an YAXArray with generic dimension names. If you would like to have meaningful axes names you can call the constructor with a Tuple of DimensionalData Dimensions along side the array.

**Reply by kongdd** [reply]
Thank you.
